### ISSUE TITLE
Document ScanCode JSON ouput format

### ISSUE THREAD

**Issue by AvishrantsSh** [nexB/scancode-toolkit#2596]
The format in which scancode.io accepts the input from pipelines is not very clear. Right now, scancode.io is built to support output generated by scancode-toolkit. Also, it seems that scancode.io uses just a handful of those results, making it even harder to figure out. Besides this, other tools and pipelines may or may not be that descriptive in their output and some sort of data marshalling may be required.
So, I think that we should decide on a uniform and consistent structure that will be acceptible by scancode.io, which can serve as a template for other pipelines.

**Comment by AyanSinhaMahapatra** [comment]
See also #2008 #2425

**Comment by chinyeungli** [comment]
@pombredanne @AyanSinhaMahapatra  Is this issue regarding the addition of references or notes indicating that the generated JSON formatted output file can be used with tools like ScanCode.io, ScanCode Workbench, and others?

**Comment by AyanSinhaMahapatra** [comment]
@chinyeungli yes we eventually want to generate this docs from the code/dosctrings entirely and this is better tracked and has more details in https://github.com/aboutcode-org/scancode-toolkit/issues/2008, so closing this as duplicate.
